cli/src/commands/diff.rs

```diff
@@ -23,6 +23,10 @@ use crate::ui::Ui;
 #[derive(clap::Args, Clone, Debug)]
 pub(crate) struct DiffArgs {
     /// Show changes in this revision, compared to its parent(s)
+    ///
+    /// If the revision is a merge commit, this shows changes between (*from*)
+    /// the automatic merge of the contents of all of its parents and (*to*)
```

Comment:
nit: remove `between`/`and` or `from`/`to`?

Reply:
I guess I'll leave `*from*` and `*to*`. It sounds a bit more awkward to me than "between", but it matches the `--from` and `--to` options.